src/views/Home/index.test.tsx

```diff
@@ -113,14 +115,25 @@ describe('The home page', () => {
             }
           });
           let component: any;
+          const mocks = [
+            {
+              request: {
+                query: GetRequestsQuery,
+                variables: { first: 20 }
+              },
+              result: {}
+            }
+          ];
           await act(async () => {
             component = mount(
               <MemoryRouter initialEntries={['/']} initialIndex={0}>
-                <Provider store={store}>
-                  <MessageProvider>
-                    <Home />
-                  </MessageProvider>
-                </Provider>
+                <MockedProvider mocks={mocks}>
```

Comment:
I'm not clear on what this mock is doing for you here.  Would you want to mock a returned query with results of a list of governance requests, and then do an `expect` to make sure they show up on the page?